src/config.rs

```diff
@@ -393,6 +394,7 @@ impl Default for DistConfig {
             cache_dir: default_dist_cache_dir(),
             toolchains: Default::default(),
             toolchain_cache_size: default_toolchain_cache_size(),
+            rewrite_includes_only: true,
```

Comment:
I'm inclined to say we should default this to `false` so we can get this merged until we know exactly how many of our users are going to have issues.

Reply:
Well, I am not sure if we are going to find that out when defaulting to false. We can default to true initially and when it turns out that too many users need to disable and cannot update their glibc, we can still change the default. A note should be added to the docs and release notes though.

Reply:
Works for me.